### DISCUSSION TITLE
spyder in anaconda mi da un errore e non parte

### DISCUSSION THREAD

**Discussion by christian-76** [community/community#53294] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

quando lancio spyder mi visualizza questo errore

![image](https://user-images.githubusercontent.com/58855813/233349194-22706968-221b-4960-ad66-d45e7663232b.png)

**Comment by queenofcorgis** [comment]
Hi @christian-76 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
